### PR TITLE
add release_date

### PR DESCRIPTION
**Pull request by andriisoldatenko**

### CODE REVIEW

File: src/astronomer-platform [hunk 1]
```diff
@@ -5,6 +5,7 @@
       "version": "0.11.1",
       "level": "bug_fix",
       "url": "https://github.com/astronomer/astronomer/releases/tag/v0.11.1"
```
Comment (ashb) [inline src/astronomer-platform]:
```suggestion
      "url": "https://github.com/astronomer/astronomer/releases/tag/v0.11.1",
```

**Comment by andriisoldatenko** [comment]
@ashb PTAL

**Comment by ashb** [comment]
> and add it to the required fields in schema.json

**Comment by andriisoldatenko** [comment]
done.

**Comment by ashb** [comment]
The error isn't hugely helpful on what's failing :(

**Comment by andriisoldatenko** [comment]
fixed.

**Pull request merged**